### PR TITLE
protocols/floodsub/: Propagate messages only to target peers

### PR DESCRIPTION
**Pull request by vnermolaev**
Superceeds #2357 

PS: I am sorry I accidentally got into a mess with my fork.

### CODE REVIEW

File: protocols/floodsub/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 0.34.0 [unreleased]
+
+- Propagate messages only to the target peers and not all connected peers.
+
 # 0.33.0 [unreleased]
```
Comment (mxinden) [inline protocols/floodsub/CHANGELOG.md]:
```suggestion
# 0.33.0 [unreleased]

- Propagate messages only to the target peers and not all connected peers (see [PR2360]).

[PR 2360]: https://github.com/libp2p/rust-libp2p/pull/2360/
```

**Review by mxinden — COMMENTED** [review]
Looks good to me otherwise.

File: protocols/floodsub/CHANGELOG.md (hunk 1 above)
Comment (vnermolaev) [reply protocols/floodsub/CHANGELOG.md]: done

**Review by mxinden — APPROVED** [review]
Thanks @vnermolaev.

**Pull request merged**